queue_services/entity-filer/src/entity_filer/filing_processors/correction.py

```diff
@@ -48,7 +48,8 @@ def process(correction_filing: Filing, filing: Dict, filing_meta: FilingMeta, bu
             staff_id=correction_filing.submitter_id
         )
     )
-    if business.legal_type in ['SP', 'GP', 'BC', 'BEN', 'CC', 'ULC']:
+    if business.legal_type in ['SP', 'GP', 'BC', 'BEN', 'CC', 'ULC'] or \
+            filing['correction']['correctedFilingType'] == 'specialResolution':
```

Comment:
Can you add 'CP' into the array instead of checking correctedFilingType?

Reply:
Take a look at the long comment thread above.  I had asked the same thing. 

it's a problem because COOP ARs, COAs and CODs would get processed and marked as `COMPLETED`.  It needs to be marked as `PENDING_CORRECTION` as they are local filings.

Reply:
I changed with checking correctedFilingType cause it's only a correction to a SR.
We had a discussion here: https://github.com/bcgov/lear/pull/2089#discussion_r1234254875